targets/native/src/animated.ts

```diff
@@ -19,9 +19,10 @@ export type WithAnimated = {
 } & AnimatedPrimitives
 
 /** The type of an `animated()` component */
-export type AnimatedComponent<
-  T extends ElementType
-> = ForwardRefExoticComponent<AnimatedProps<ComponentPropsWithRef<T>>>
+export type AnimatedComponent<T extends ElementType> =
+  ForwardRefExoticComponent<
+    AnimatedProps<ComponentPropsWithRef<T>> & { children: ReactNode }
```

Comment:
This change seems to have broken the following code for me:

```js
<animated.View style={{ opacity }} onTouchEnd={handleDismiss} />
```

It now gives this error:

```text
error TS2741: Property 'children' is missing in type '{ style: { opacity: SpringValue<number>; }; onTouchEnd: () => void; }' but required in type '{ children: ReactNode; }'.
```

I think that this should have been:

```suggestion
    AnimatedProps<ComponentPropsWithRef<T>> & { children?: ReactNode }
```

Reply:
Indeed, could you open a PR?

Reply:
Yeah I should probably have linked that one, sorry 😅 

https://github.com/pmndrs/react-spring/pull/1925

It's merged now 🚀